variables.tf

```diff
@@ -16,6 +16,17 @@ variable "oauth_client_secret" {
   sensitive   = true
 }
 
+variable "log_level" {
+  description = "Log level"
+  default     = "ERROR"

```

Comment:
As above, I'd probably default to lower than ERROR - INFO or WARNING feel like they might well provide context you find useful when looking back at your logs after an issue